NuclearData/xsPackages/neutronXsPackages_class.f90

```diff
@@ -22,6 +22,7 @@ module neutronXsPackages_class
   !!   capture          -> sum of all reactions without secendary neutrons excluding fission [1/cm]
   !!   fission          -> total Fission MT=18 Cross-section [1/cm]
   !!   nuFission        -> total average neutron production Cross-section [1/cm]
+  !!   energyDepoZero   -> energy deposition, for mode zero.  [MeV/cm]
```

Comment:
I would be inclined to rename this property. The reference to mode 0 is (I think) unique to Serpent, whereas the fission cross section * the Q value is commonly understood. Maybe qFission in analogy with nuFission would be more clear?